docs/api/cases/cases-api-find-connectors.asciidoc

```diff
@@ -0,0 +1,58 @@
+[[cases-api-find-connectors]]
+== Find connectors API
+++++
+<titleabbrev>Find connectors</titleabbrev>
+++++
+
+Retrieves information about <<action-types,connectors>>.
+
+In particular, only the connectors that are supported for use in cases are
+returned. Refer to the list of supported external incident management systems in
+<<add-case-connectors>>.
+
+=== Request
+
+`GET <kibana host>:<port>/s/<space_id>/api/cases/configure/connectors/_find`
+
+=== Prerequisite
+
+You must have `read` privileges for the *Actions and Connectors* feature in the
```

Comment:
I think we should do https://github.com/elastic/kibana/pull/127686/files#r829225831 to all of our endpoints.